scripts/views/eventView.js

```diff
@@ -9,10 +9,30 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
-    if (zip) {
-      TownHall.lookupZip($('#look-up input').val());
-      eventHandler.resetFilters();
+    if (zip && zip.length === 5) {
+      repZip = zip;
+    } else if (zip && zip.length > 5) {
+      zip = zip.match(/\d{5}/g)[0];
+      plusFour = zip.match(/[+-\s]\d{4}/g);
```

Comment:
It looks like this line allows for the 5+4 zip to be separated by a + or - or whitespace character.  Everything I found indicated that the format only allows for a dash, e.g. 12345-6789